webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/zarr3/Zarr3ArrayHeader.scala

```diff
@@ -248,5 +249,41 @@ object Zarr3ArrayHeader extends JsonImplicits {
         "storage_transformers" -> zarrArrayHeader.storage_transformers,
         "dimension_names" -> zarrArrayHeader.dimension_names
       )
+
+  }
+  def fromDataLayerToVersion3(dataLayer: DataLayer): Zarr3ArrayHeader = {
```

Comment:
since this is a member of object Zarr3ArrayHeader, I’d say `fromDataLayer` is sufficient as a name, what do you think?